src/components/organisms/UiPhoneNumber/_internal/UiPhoneNumberPrefix.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <UiDropdown
+    class="ui-phone-number__dropdown"
+    :model-value="modelValue"
+    :popover-attrs="{ class: 'ui-phone-number__dropdown-popover' }"
+    @update:model-value="handleOnChangeCode"
+  >
+    <template
+      #toggle="{
+        toggleHandler, isOpen,
+      }"
+    >
+      <UiButton
+        id="ui-phone-number__dropdown"
+        class="ui-button--outlined ui-button--has-icon ui-phone-number__dropdown-button"
+        :class="{ 'ui-button--has-error': error }"
```

Comment:
```suggestion
        :class="[
          'ui-button--outlined ui-button--has-icon ui-phone-number__dropdown-button',
          { 'ui-button--has-error': error }
        ]"
```

Reply:
and I can find the prefix error; here you have error without red prefix input
![Zrzut ekranu 2023-12-19 o 14 45 48](https://github.com/infermedica/component-library/assets/12138170/3db49f12-b683-495d-9f52-7d7c8d7d4723)